src/test/java/se/sundsvall/webmessagecollector/configuration/ShedlockUsageTest.java

```diff
@@ -70,10 +71,11 @@ private Map<String, List<Method>> getMethodsAnnotatedWith(final BeanDefinition c
 				// move to the upper class in the hierarchy in search for more methods
 				klazz = klazz.getSuperclass();
 			}
-			return Map.of(candidate.getBeanClassName(), methods);
-		} catch (ClassNotFoundException e) {
+			return Map.of(Objects.requireNonNull(candidate.getBeanClassName()), methods);
```

Comment:
Is the "Objects.requireNonNull" necessary? 

The we pass an object that is null and NPE is thrown either way, with or without it.

Reply:
Yes.